frontend/src/reducers/appReducer.js

```diff
@@ -0,0 +1,15 @@
+import { CHANGE_VIEW } from '../actions/actionsTypes';
+import initialState from './initialState';
+
+export default (state = initialState, action) => {
```

Comment:
Naming of this file is a little confusing. Could you name all reducers individual to their functions?